autogpt/commands/file_operations.py

```diff
@@ -133,7 +133,8 @@ def write_to_file(filename: str, text: str) -> str:
         str: A message indicating success or failure
     """
     if check_duplicate_operation("write", filename):
-        return "Error: File has already been updated."
+        append_to_file(filename, text)
+        return "Append to file complete!"
```

Comment:
This isn't related to a naming convention, is it? Submit as a separate PR

Reply:
I added it in the description 'Resolved an issue with the file logger code I had, to ensure we append if we already have written.'